CMakeLists.txt

```diff
@@ -1078,6 +1078,10 @@ zephyr_compile_options(
   $<$<COMPILE_LANGUAGE:C>:-std=${CSTD}>
 )
 
+configure_file(
+     $ENV{ZEPHYR_BASE}/include/arch/arm/cortex_m/scripts/linker_section_alignment.ld
```

Comment:
You don't need a configure_file.

At least I don't see anything in the linker script that requires it.

Reply:
done as discussed offline; added the linker symbol directly in linker.ld, removed the introduced file and the changes in CMakeLists.txt